scripts/init.d-xapi-domains

```diff
@@ -6,6 +6,12 @@
 # chkconfig: 345 99 00
 # description: Start/stop XE VMs
 
+### BEGIN INIT INFO
+# Required-Start:    $remote_fs
+# Short-Description: Start/stop XE VMs
+# Description: Stop domains automatically when domain 0 shuts down.
```

Comment:
Unfortunately sysv initscripts have some "special" features where comments are parsed for extra information. There's some details here: https://wiki.debian.org/LSBInitScripts